network/common/pom.xml

```diff
@@ -55,6 +55,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
+      <scope>compile</scope>
```

Comment:
Isn't compile the default scope? i.e. couldn't you leave out the explicit `compile` scope here and just not specify a scope?